front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java

```diff
@@ -145,15 +168,15 @@ public void setReleaseByVersion(String version, Release release) {
      * @return Whether or not the plugin supports the given service
      */
     public boolean supportsService(@Nonnull String service) {
-      return Arrays.stream(requires.split(","))
-          .anyMatch(
-              it -> {
-                Matcher m = SUPPORTS_PATTERN.matcher(it.trim());
-                if (m.matches()) {
-                  return m.group(SUPPORTS_PATTERN_SERVICE_GROUP).equals(service);
-                }
-                return false;
-              });
+      return getParsedRequires().stream().anyMatch(
+        it -> it.getService().equalsIgnoreCase(service));
     }
   }
+
+  @Data
+  public static class ServiceRequirement {
```

Comment:
this ok here?  Probably should move it to the Event class...

Reply:
Nah, it's fine here.